src/services/utils.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { getUserToken } from '../redux/commonStore.ts';
+
+export const backendFetch = (url: string, init: any, token?: string) => {
+    const initCopy = prepareRequest(init, token);
+    return safeFetch(url, initCopy);
+};
+
+const prepareRequest = (init: any, token?: string) => {
+    if (!(typeof init == 'undefined' || typeof init == 'object')) {
+        throw new TypeError(
+            'Argument 2 of backendFetch is not an object : ' + typeof init
+        );
+    }
+    const initCopy = Object.assign({}, init);
+    initCopy.headers = new Headers(initCopy.headers || {});
+    initCopy.headers.append(
+        'Authorization',
+        'Bearer ' + token ?? getUserToken()
```

Comment:
Why `token ?? getUserToken()` ? I think we just want to use getUserToken and remove all the tokens in argument no ?

Reply:
As it's supposed to be a temporary solution, I kept my work with the token in functions args and made it optional, in case we need it later.
But I can revert all this.

Reply:
It's also how it's done in gridexplore / gridstudy.
https://github.com/gridsuite/gridexplore-app/blob/a01ac4c3d1804830a062e7d897be6dc39b60d0b2/src/utils/rest-api.js#L112

Reply:
yeah it's a bit weird to expose this argument in all the APIs but then no components should use it because it's done in the 'authentication layer'. I would rather remove it, but as it was in the apps already, it's up to you.

Reply:
Breifly discussed with @jonenst , I think we will keep it